doc/chapter-password.txt

```diff
@@ -0,0 +1,31 @@
+When using external APIs for newsfeeds, the user has different options for how
+to provide the password to authenticating against the API.
```

Comment:
It should be either "to authenticate" (against the API), "for authentication" or "for authenticating".